pkg/subctl/benchmark/latency.go

```diff
@@ -117,5 +128,31 @@ func runLatencyTest(f *framework.Framework, testParams benchmarkTestParams) {
 		nettestClientPod.Pod.Name, clusterAName, nettestClientPod.Pod.Spec.NodeName, remoteIP))
 
 	framework.By(fmt.Sprintf("Waiting for the client pod %q to exit, returning what client sent", nettestClientPod.Pod.Name))
-	nettestClientPod.AwaitFinishVerbose(Verbose)
+	latency := nettestClientPod.AwaitFinishVerbose(Verbose)
```

Comment:
The data you want can be accessed via `nettestClientPod.TerminationMessage` w/o explicitly returning it.

Reply:
Didn't know that. Made necessary changes.